core/conventional-commits/__tests__/conventional-commits.test.js

```diff
@@ -481,6 +482,42 @@ describe("conventional-commits", () => {
       `);
     });
 
+    it("supports config builder presets", async () => {
+      const cwd = await initFixture("fixed");
+
+      const configForPresetNameString = await getChangelogConfig("./scripts/config-builder-preset", cwd);
+      expect(configForPresetNameString).toBeDefined();
+
+      const presetConfigObject = { name: "./scripts/config-builder-preset", key: "value" };
+      const configForPresetConfigObject = await getChangelogConfig(presetConfigObject, cwd);
+
+      expect(configForPresetConfigObject).toBeDefined();
+      expect(configForPresetConfigObject.key).toBe(presetConfigObject.key);
+
+      await gitTag(cwd, "v1.0.0");
+
+      const [, pkg2] = await getPackages(cwd);
+
+      // make a change in package-2
+      await pkg2.set("changed", 1).serialize();
+      await gitAdd(cwd, pkg2.manifestLocation);
+      await gitCommit(cwd, "fix(pkg2): A commit using a legacy callback preset");
```

Comment:
This commit message looks like copypasta, but I suppose all it does is look confusing in the test assertion, not invalid. I can fix later.